.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
+- [ ] Updated documentation
+- [ ] External dependency updates
+- [ ] Internal dependency updates (with link to diff)
```

Comment:
What is this?

Reply:
we have a policy that if a change is made to an internal dependency (e.g. iceqube or perseus renderer) we need to link to the diff

Reply:
Thanks, that makes sense, especially when external updates are still pinned as commits hashes.